src/test/java/teammates/test/cases/ui/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -78,6 +78,9 @@ private void testLinks() {
     }
 
     private void testContent() {
+
+        AppPage.logout(browser);
```

Comment:
I think we should put this line after **______TS("unreg student");** as this is needed for that test specifically. If someone moves that test, this needs to move as well